docs/release-notes.md

```diff
@@ -1,101 +1,81 @@
- <iframe width="800" height="450" src="https://www.youtube.com/embed/gTDXOdsCRZk" title="Open AMT v2.5 Release Video" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+<div style="text-align:center;">
+ <iframe width="800" height="450" src="https://www.youtube.com/embed/gTDXOdsCRZk" title="Open AMT November Release Video" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 </div>
-<br> -->
+<br>
 
 !!! note "Note From the Team"
     Hey everyone,
 
-    Very quick and small release this week to deliver some new features and fixes!  Going forward, we'll be moving to a monthly cadence for Open AMT Cloud Toolkit with sub components releasing when they have features or fixes ready.  Our hope is that users of the Open AMT Cloud Toolkit will see the features they need delivered even faster.  In order to manage the overhead involved with this faster release cadence, we'll be moving to a monthly roll-up and release announcement starting at the end of November where we will summarize all of the releases during the previous month.  You can find the details of this release under [what's new](#whats-new) section.
+    For those in the United States, I hope you all had a wonderful Thanksgiving holiday and for our customers everywhere, our team is very thankful for your feedback, contributions, and the opportunity to continue working with all of you.  You all are the reason we are able to continue working on Open AMT Cloud Toolkit.  We have implemented some much asked for improvements and features in our November release in addition to a whole bevy of resolved issues and external contributions!  Check out the [release video](#release-highlights) where Mike talks about this release's highlights and you can find the details of this release under [what's new](#whats-new) section.
 
     *Best wishes,*  
     *The Open AMT Cloud Toolkit Team*
 
 
 ## What's New?
 
+:material-star:** Database update required **
+An upgrade to the `mpsdb` database is required. Please run the following SQL script to add two new columns before upgrading the services:
+``` sql
+ALTER TABLE devices 
+ADD COLUMN IF NOT EXISTS friendlyname varchar(256);
+ALTER TABLE devices 
+ADD COLUMN IF NOT EXISTS dnssuffix varchar(256);
+```
+
 :material-new-box:** Feature: AMT Maintenance commands **
 
-Three new maintenance commands have been added to RPC and RPS to support syncing the AMT Clock, syncing the IP settings, and changing the AMT password.
+We have added a maintenance task that will enable syncing the OS Hostname and DNS Suffix with AMT Hostname and DNS Suffix.  When this command is run, if the OS and AMT host names are different, the OS hostname will be applied to AMT.  This enables AMT to respond to network requests that use the fully qualified domain name (FQDN) of the device.
```

Comment:
We added a maintenance task command that enables syncing the OS hostname and DNS Suffix with AMT hostname and DNS suffix.  If the OS and AMT hostnames are different when the command is run, the OS hostname will be applied to AMT.  This maintenance task enables AMT to respond to network requests that use the fully qualified domain name (FQDN) of the device.

Hey @matt-primrose, do you want to mention the name of the command here?